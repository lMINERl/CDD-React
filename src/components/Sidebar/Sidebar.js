import React from 'react';
const Sidebar = props => {
  return (
    <ul className="sidebar navigation__logo">
      <li className="sidebar__row sidebar__row--dark">
        <i className="fa fa-dashboard"></i>
        <span className="sidebar__text">Dashboard</span>
        <i className="sidebar__row--extend fa fa-angle-right"></i>
      </li>
      <li className="sidebar__row sidebar__row--dark">
        <i className="fa fa-music"></i>
        <span className="sidebar__text">Mail</span>
        <i className="sidebar__row--extend fa fa-angle-right"></i>
      </li>
      <li className="sidebar__row sidebar__row--dark">
        <i className="fa fa-server"></i>
        <span className="sidebar__text">Database</span>
        <i className="sidebar__row--extend fa fa-angle-right"></i>
      </li>
      <li className="sidebar__row sidebar__row--dark">
        <i className="fa fa-bar-chart"></i>
        <span className="sidebar__text">Analysis</span>
        <i className="sidebar__row--extend fa fa-angle-right"></i>
      </li>
    </ul>
  );
};

export default Sidebar;
