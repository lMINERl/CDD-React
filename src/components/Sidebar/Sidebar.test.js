import { React, shallow } from '../../test/Configurations';
import Sidebar from './Sidebar';

describe('<Sidebar />', () => {
  test('renders', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper).toMatchSnapshot();
  });
});
