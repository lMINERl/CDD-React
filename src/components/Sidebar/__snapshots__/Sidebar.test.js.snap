// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> renders 1`] = `
<ul
  className="sidebar navigation__logo"
>
  <li
    className="sidebar__row"
  >
    <i
      className="fa fa-dashboard"
    />
    <span
      className="sidebar__text"
    >
      Dashboard
    </span>
    <i
      className="sidebar__row--extend fa fa-angle-right"
    />
  </li>
  <li
    className="sidebar__row"
  >
    <i
      className="fa fa-music"
    />
    <span
      className="sidebar__text"
    >
      Mail
    </span>
    <i
      className="sidebar__row--extend fa fa-angle-right"
    />
  </li>
  <li
    className="sidebar__row"
  >
    <i
      className="fa fa-server"
    />
    <span
      className="sidebar__text"
    >
      Database
    </span>
    <i
      className="sidebar__row--extend fa fa-angle-right"
    />
  </li>
  <li
    className="sidebar__row"
  >
    <i
      className="fa fa-bar-chart"
    />
    <span
      className="sidebar__text"
    >
      Analysis
    </span>
    <i
      className="sidebar__row--extend fa fa-angle-right"
    />
  </li>
</ul>
`;
